Makefile

```diff
@@ -15,8 +15,16 @@ GOLANG_VERSION?="1.16"
 GO ?= $(shell source ./scripts/common.sh && build::common::get_go_path $(GOLANG_VERSION))/go
 GO_TEST ?= $(GO) test
 
+ifdef CODEBUILD_SOURCE_VERSION ## If in codebuild environment, use the branch-specific bundle manifest URL for non-main (e.g. release branch) sources
+ifeq (,$(findstring $(CODEBUILD_SOURCE_VERSION),main))
+BUNDLE_MANIFEST_URL?=https://dev-release-prod-pdx.s3.us-west-2.amazonaws.com/${CODEBUILD_SOURCE_VERSION}/bundle-release.yaml
+$(info    Using branch-specific BUNDLE_RELEASE_MANIFEST_URL $(BUNDLE_MANIFEST_URL))
+endif
+endif
+
 RELEASE_MANIFEST_URL?=https://dev-release-prod-pdx.s3.us-west-2.amazonaws.com/eks-a-release.yaml
 BUNDLE_MANIFEST_URL?=https://dev-release-prod-pdx.s3.us-west-2.amazonaws.com/bundle-release.yaml
```

Comment:
Im not 100% sure here if this will override it being set above?  I know the ? should stop that for env vars and vars pass on command line, but not 100% sure about vars within the make?

Reply:
tested it, won't override it